content/posts/why-sre/index.mdx

```diff
@@ -0,0 +1,159 @@
+Also, the management team had previous experience with many of the SRE principles and practices.
+
+One the biggest motivating factors for this team is pain.
+Both customer pain and the team's pain.
+An example of customer pain would be not being able to purchase and deploy at the press of a button.
+The team saw this from the customers perspective and looked at ways of automating away the manual steps and engagements.
+They identified areas where a deploy could fail late in the process and someone would have to manually reset and trigger things again.
+The solution here was adding pre-installation checks.
+It's a reactive model, but it's also [actively managing TOIL early on](https://github.com/operate-first/sre/blob/main/sre_maturity.md#toil-management-and-operational-improvements).
+The concept of tying your decisions to what the customer cares about and measuring it is an important one for this team.
+They are establishing Service Level Indicators (SLIs) and Objectives (SLOs) in an effort to be more proactive.
+
+The management team deliberately hired from a diverse group of backgrounds.
+The focus was on hiring people who could lead & speak their mind.
+People who could think from the bottom up, as management wasn't going to have all the ideas.
+They recognised the importance of a good education plan.
+When hiring for the team it was always going to be a challenge to get everyone up to the same level of confidence.
+Some people have a lot of Ansible skills, while some have a lot of Azure skills.
+It's important to invest in the team by recommending courses and allowing time for people to upskill.
+
+## Case Study 3: Services on OpenShift
+
+There's an SRE team in Red Hat called the SRE Services (or SRES) team.
+At the time of writing they manage roughly 50 services.
+These services range from customer facing services to ancillary services that support those, all of which run on OpenShift.
+The team uses a gitops model to normalise things across services and manage building, deploying, configuring & upgrading things.
+This model allows teams to build a service and onboard them to this central SRE team by following standardised processes and using automation.
+However, it took a while to get to this point. This SRES team was a natural evolution of a bigger plan.
+
+Building on the Managed OpenShift SRE story in Case Study 1, the OpenShift platform reached a high level of reliability, maturity and scale.
+There was a transition point to start introducing services to the platform.
+The SRE model was a success already so it made sense to use that same model to solve this new problem.
+The SRES team was formed and tasked with solving that problem.
+
+Git was central to change management for this team.
+Everything went through git.
+Pull requests had checks and automated pipelines for rolling out changes when they merged.
+Configuration was declarative and the service contract was clearly defined in the versioned json schema.
```

Comment:
```suggestion
Configuration was declarative and the service contract was clearly defined in the [versioned json schema](https://github.com/app-sre/qontract-schemas/tree/main/schemas).
```

Reply:
I'm trying to confirm if :this: is the 100% correct link. If you're in a hurry to merge, this suggestion can be dropped.

Reply:
Got an ack, seems it's the correct URL.